src/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { createAppContainer, createSwitchNavigator } from 'react-navigation'
+import NavigationService from './navigationService'
+import SignUp from './screens/SignUp'
+import Home from './screens/Home'
+import LoadingScreen from './screens/LoadingScreen'
+import * as firebase from 'firebase'
+import firebaseConfig from './configs/firebase'
+
+const initializeFirebase = () => {
+  if (!firebase.apps.length) {
+    firebase.initializeApp(firebaseConfig)
+  }
+}
+
+initializeFirebase()
```

Comment:
Do you need to call this function again somewhere else? If you don't, then you can write the `if` statement at the top level, without a function. It'll be executed only once, the first time this module is imported by another one (or run as an entry point).